schemas/zora/20210604.json

```diff
@@ -0,0 +1,92 @@
+{
+  "description": "This schema describes version 20210604 of the Zora Metadata Standard",
+  "title": "Zora20210604",
+  "$id": "https://zora.co/schemas/ZoraMetadata.json",
+  "$defs": {
+    "attribute": {
+      "type": "object",
+      "required": ["trait_type", "value"],
+      "properties": {
+        "trait_type": {
+          "type": "string",
+          "description": "The name of the trait"
+        },
+        "value": {
+          "type": ["string", "number", "boolean"],
+          "description": "The value of the trait"
+        },
+        "display_type": {
+          "type": "string",
+          "description": "A field indicating how the `value` data should be displayed. Defaults to 'string'"
+        }
+      }
+    }
+  },
+  "type": "object",
+  "properties": {
+    "description": {
+      "description": "The description of the media",
+      "examples": [
+        "This paper describes protocol to create, share and exchange universally accessible and valuable information on the internet."
+      ],
+      "type": "string"
+    },
+    "mimeType": {
+      "description": "The mimeType of the media",
+      "examples": [
+        "text/plain"
+      ],
+      "type": "string"
+    },
+    "name": {
+      "description": "This property is the name of the Media",
+      "examples": [
+        "Zora Whitepaper"
+      ],
+      "type": "string"
+    },
+    "version": {
+      "description": "This property defines the calendar version of the schema so that consumers can correctly parse the json",
+      "examples": [
+        "zora-20210604"
+      ],
+      "type": "string"
+    },
+    "image": {
```

Comment:
Add in animation url to support animation previews for dynamic content / rendering support?

Reply:
👍  done